src/lib/components/artwork_grids/infinite_scroll_grid.tsx

```diff
@@ -35,52 +28,56 @@ export const PageEndThreshold = 1000
  *   - the calculation currently only takes into account the size of the image, not if e.g. the sale message is present
  */
 
-type Artworks = Array<{
-  __id: string,
-  image: {
-    aspect_ratio: number | null,
-  } | null,
-}>
+type Artworks = Array<
+  {
+    __id: string,
+    image:
+      | {
+        aspect_ratio: number | null,
+      }
+      | null,
+  }
+>
 
 interface Props extends ArtistRelayProps, GeneRelayProps {
   /** The direction for the grid, currently only 'column' is supported . */
-  sectionDirection: string;
```

Comment:
@orta we settled on no commas in reaction-force, if you had legitimate issues with that then please elaborate as to what they were.